selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CheckFactoryWithMultiModuleTest.java

```diff
@@ -44,6 +45,7 @@
   private String gitHubUsername;
 
   @Inject private SeleniumWebDriver seleniumWebDriver;
```

Comment:
**seleniumWebDriver** looks redundant in class.

Reply:
SeleniumWebDriver is needed in some factory tests for factory url opening(TestFactory.open(seleniumWebDriver)).

Reply:
Looks like an issue. It seems we can easily reuse field `TestFactory#seleniumWebDriver` instead of argument `seleniumWebDriver`.